src/helpers/stateHelpers.js

```diff
@@ -102,3 +102,9 @@ export const sortCommentsFromSteem = (list, commentsState, sortBy = 'trending')
     )
     .reverse();
 };
+
+export const createAsyncActionType = type => ({
+  PENDING: `${type}_PENDING`,
```

Comment:
We use `START`, `SUCCESS`, `ERROR` for different states of action progress: https://github.com/busyorg/busy/blob/7b122f361e3923cd3e18016659b25b990bb8c68e/src/store.js#L26

Reply:
@Sekhmet ahh I see now... since it goes along with the `redux-promise-middleware`, updated it to include the action